src/index.html

```diff
@@ -3,12 +3,30 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet"
+    crossorigin="true">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="logo"><img src="images/logo.png" alt="MOYO"></div>
+        <nav class="navbar">
+          <ul class="navbar_list">
+            <li><a href="#" class="is-active aaa">APPLE</a></li>
```

Comment:
```suggestion
            <li><a href="#" class="is-active">APPLE</a></li>
```
why do you need this strange class `aaa`?